node/src/chain_spec.rs

```diff
@@ -161,6 +161,10 @@ fn eden_testnet_genesis(
 
 		// DAO
 		dao_reserve: Default::default(),
+
+		polkadot_xcm: PolkadotXcmConfig {
+			safe_xcm_version: Some(2),
```

Comment:
It's better to put 
```
const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
```
At the top of this file in a suitable place and then change the highlighted line as follows:
```suggestion
			safe_xcm_version: Some(XCM_VERSION),
```